pkg/crc/constants/constants.go

```diff
@@ -83,31 +83,31 @@ func GetAdminHelperURL() string {
 	return GetAdminHelperURLForOs(runtime.GOOS)
 }
 
-func defaultBundleForOs(preset crcpreset.Preset) map[string]string {
+func defaultBundleForOs(preset crcpreset.Preset, bundleVersion string) map[string]string {
```

Comment:
As soon as you pass it a version, it's no longer the `default` bundle for the OS.

Reply:
I will create an helper  `bundleForOs`  so that can be used with `defaultBundleForOs`.